duckduckgo/macos-browser
# 3 of 3 . Home Button (Pixels)

Task/Issue URL: https://app.asana.com/0/72649045549333/1203345455295029/f

**Description**:
Adds pixels to track home button usage

**Steps to test this PR**:
1. Enable to home button
2. Confirm `m_mac_enable_home_button` pixel is fired
3. Disable Home button 
4. Confirm `m_mac_disable_home_button` pixel is fired
5. Set a custom home page
6. Confirm `m_mac_set_new_homepage` pixel is fired



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
